pkg/pxc/users/users.go

```diff
@@ -0,0 +1,98 @@
+package users
+
+import (
+	api "github.com/percona/percona-xtradb-cluster-operator/pkg/apis/pxc/v1"
+	"github.com/percona/percona-xtradb-cluster-operator/pkg/pxc/app"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+var log = logf.Log.WithName("users-manager")
+
+func Job(cr *api.PerconaXtraDBCluster) *batchv1.Job {
+	labels := make(map[string]string)
+	for key, value := range cr.Spec.Users.Labels {
+		labels[key] = value
+	}
+	labels["type"] = "user-manager"
+	labels["cluster"] = cr.Name
+	labels["job-name"] = cr.Name + "-pxc-user-manager"
```

Comment:
what if we need to apply different secrets simultaneously?
will it work fine with the same job name?

Reply:
done https://github.com/percona/percona-xtradb-cluster-operator/pull/342/commits/3417c02c2c6246ae92274a0c62f72501f6937a3b